tests/test_config.py

```diff
@@ -1,3 +1,13 @@
+"""Tests for the configs provided by the electricitymap.contrib.config
+package.
+
+The values in these configs should be loaded from the config files in the
+config/ directory. The tests in this file ensure that the values are loaded.
+It doesn't check all or most values, but serves as a sanity check that the
+configs are loaded correctly.
+"""
+
+
 import json
```

Comment:
This is unused. Along with DeepDiff further down.

Reply:
Done, fixed.